### ISSUE TITLE
artifacts-ubuntu2004 test fails with - sudo: yum: command not found

### ISSUE THREAD

**Issue by yarongilor** [scylladb/scylla-cluster-tests#3396]
error log:
```
2021-04-12 15:35:47.890: (TestFrameworkEvent Severity.ERROR), source=ArtifactsTest.SetUp()
exception=[Node artifacts-ubuntu2004-jenkins-db-node-2d927cea-0-1 [35.231.29.146 | 10.142.0.55] (seed: True)] NodeSetupFailed: Encountered a bad command exit code!
Command: 'bash -cxe "\nsudo yum install -y cyrus-sasl\nsudo systemctl enable saslauthd\necho \'MECH=ldap\' | sudo tee -a /etc/sysconfig/saslauthd\nsudo touch /etc/saslauthd.conf\necho \'saslauthd_socket_path: /run/saslauthd/mux\' | sudo tee -a /etc/scylla/scylla.yaml\n"'
Exit code: 1
Stdout:
Stderr:
+ '[' -z '' ']'
+ return
+ case $- in
+ return
+ sudo yum install -y cyrus-sasl
sudo: yum: command not found
Traceback (most recent call last):
File "/jenkins/slave/workspace/scylla-master/artifacts/artifacts-ubuntu2004-test/scylla-cluster-tests/sdcm/cluster.py", line 3457, in node_setup
cl_inst.node_setup(_node, **setup_kwargs)
File "/jenkins/slave/workspace/scylla-master/artifacts/artifacts-ubuntu2004-test/scylla-cluster-tests/sdcm/cluster.py", line 4082, in node_setup
node.remoter.run('bash -cxe "%s"' % setup_script)
File "/jenkins/slave/workspace/scylla-master/artifacts/artifacts-ubuntu2004-test/scylla-cluster-tests/sdcm/remote/remote_base.py", line 604, in run
result = _run()
File "/jenkins/slave/workspace/scylla-master/artifacts/artifacts-ubuntu2004-test/scylla-cluster-tests/sdcm/utils/decorators.py", line 61, in inner
return func(*args, **kwargs)
File "/jenkins/slave/workspace/scylla-master/artifacts/artifacts-ubuntu2004-test/scylla-cluster-tests/sdcm/remote/remote_base.py", line 596, in _run
return self._run_execute(cmd, timeout, ignore_status, verbose, new_session, watchers)
File "/jenkins/slave/workspace/scylla-master/artifacts/artifacts-ubuntu2004-test/scylla-cluster-tests/sdcm/remote/remote_base.py", line 537, in _run_execute
result = connection.run(**command_kwargs)
File "<decorator-gen-3>", line 2, in run
File "/usr/local/lib/python3.9/site-packages/fabric/connection.py", line 30, in opens
return method(self, *args, **kwargs)
File "/usr/local/lib/python3.9/site-packages/fabric/connection.py", line 721, in run
return self._run(self._remote_runner(), command, **kwargs)
File "/usr/local/lib/python3.9/site-packages/invoke/context.py", line 101, in _run
return runner.run(command, **kwargs)
File "/usr/local/lib/python3.9/site-packages/invoke/runners.py", line 291, in run
return self._run_body(command, **kwargs)
File "/usr/local/lib/python3.9/site-packages/invoke/runners.py", line 442, in _run_body
raise UnexpectedExit(result)
invoke.exceptions.UnexpectedExit: Encountered a bad command exit code!
Command: 'bash -cxe "\nsudo yum install -y cyrus-sasl\nsudo systemctl enable saslauthd\necho \'MECH=ldap\' | sudo tee -a /etc/sysconfig/saslauthd\nsudo touch /etc/saslauthd.conf\necho \'saslauthd_socket_path: /run/saslauthd/mux\' | sudo tee -a /etc/scylla/scylla.yaml\n"'
Exit code: 1
Stdout:
Stderr:
+ '[' -z '' ']'
+ return
+ case $- in
+ return
+ sudo yum install -y cyrus-sasl
sudo: yum: command not found
```
Test details
Test: artifacts_test.ArtifactsTest.test_scylla_service
Build number: 127
Backend: gce: us-east1
Kernel version: 5.4.0-1040-gcp
Test-id: 2d927cea-398b-4009-973c-13a98c185506
Start time: 2021-04-12 15:34:43
End time: 2021-04-12 15:36:08
Started by user: yarongilor
Private repo: http://downloads.scylladb.com/unstable/scylla/master/deb/unified/2021-04-11T22:10:39Z/scylladb-master/scylla.list
Test result
FAILED
System under test
ScyllaDB version:
ScyllaDB repo: http://downloads.scylladb.com/unstable/scylla/master/deb/unified/2021-04-11T22:10:39Z/scylladb-master/scylla.list
Actual gce image used: ubuntu-2004-focal-v20210325
AWS region: us-east1
Instance type: n1-standard-2
Scylla packages installed:
No scylla packages are installed. Please check log files.

**Comment by yarongilor** [comment]
Cc @bentsi . perhaps related as well to latest changes?

**Comment by roydahan** [comment]
No, it's not related.
It's a bug that @amoskong introduced today.

@amoskong the concept of installing something not in the correct place is wrong.

**Comment by amoskong** [comment]
Fixed by #3399
